ph1_weekly.py

```diff
@@ -0,0 +1,21 @@
+import os
+from decouple import config 
+from pymongo import MongoClient
+from scripts._0_2_8_human_to_adm_comparison import compare_probes
+from scripts._0_3_0_percent_matching_probes import find_matching_probe_percentage
+
+
+if __name__ == '__main__':
+    # set up mongo
+    client = MongoClient(config('MONGO_URL'))
+    db = client.dashboard
+
+    # run 'weekly' version of probe matcher
+    os.system('python3 ph1_probe_matcher.py -i ph1_sim_files -w')
```

Comment:
For me, my system python 3.x is simply `python`, so I needed to update the script.  But that's no problem-- as long as it works on prod.

Reply:
This ok on my end, I have python3 and python sym linked to python 3.9 on prod.